### PR TITLE
Request to run 2017-12 and a no-op script change.

### PR DESCRIPTION
**Pull request by kdraeder**
diags_batch.csh
> This script is not used in production mode.
> Add a line to document an example of making a script dependent
> on the successful finish of another job.

### CODE REVIEW

File: diags_batch.csh [hunk 1]
```diff
@@ -27,6 +27,8 @@
 # It's helpful to use the $casename here.
 #PBS  -o diags_batch.eo
 #PBS  -j oe 
+# Run this after st_archive finishes with no errors.
+#PBS  -W depend=afterok:2290681.chadmin1.ib0.cheyenne.ucar.edu
```
Comment (timhoar) [inline diags_batch.csh:32]: pretty sneaky

Comment (timhoar) [reply diags_batch.csh:32]: This did create a merge conflict ... your (local) diags_batch.csh does not have the -k eod 
directive and it should. Adding the dependency happened to conflict with where the -k eod
was added.

**Comment by kdraeder** [comment]
Doh!  I'll add that.

File: diags_batch.csh (hunk 1 above)
Comment (timhoar) [reply diags_batch.csh:32]: BTW ... I fixed the merge conflict on the web.

**Pull request merged**